### ISSUE TITLE
design2params crash on empty defaultssheet

### ISSUE THREAD

**Issue by berland** [equinor/semeio#145]
On a scratch dir where parameters.txt does not exist, the following crash is observed:
```
Traceback (most recent call last):
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/semeio/jobs/scripts/design2params.py", line 85, in <module>
    main(sys.argv[1:])
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/semeio/jobs/scripts/design2params.py", line 80, in main
    log_level=parsed_args.log_level,
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/semeio/jobs/design2params/design2params.py", line 51, in run
    realization, parameters, parametersfilename, design_matrix_sheet, default_sheet
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/semeio/jobs/design2params/design2params.py", line 88, in _complete_parameters_file
    pd.merge(realization_values, default_sheet, on="keys", how="outer"),
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/pandas/core/reshape/merge.py", line 47, in merge
    validate=validate)
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/pandas/core/reshape/merge.py", line 529, in __init__
    self.join_names) = self._get_merge_keys()
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/pandas/core/reshape/merge.py", line 833, in _get_merge_keys
    right._get_label_or_level_values(rk))
  File "/prog/res/komodo/2020.08.01-py36/root/lib/python3.6/site-packages/pandas/core/generic.py", line 1706, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'keys'
```

credit: cott

**Comment by berland** [comment]
Problem is due to empty DefaultSheet in design matrix.

**Comment by berland** [comment]
Fixed in #146
